tests/integration/test_headers_and_subtotals.py

```diff
@@ -1127,6 +1127,65 @@ def it_calculate_col_residuals_for_subtotals(self):
                 ],
             ],
         )
+        np.testing.assert_almost_equal(
```

Comment:
this expectation should be extracted. It's long enough to disrupt reading the test.